runtime/near-vm-runner/src/tests/contract_preload.rs

```diff
@@ -188,3 +193,35 @@ pub fn test_run_preloaded() {
     #[cfg(feature = "wasmer1_vm")]
     test_vm_runner(true, VMKind::Wasmer1, 100);
 }
+
+fn test_precompile_vm(vm_kind: VMKind) {
+    let mock_cache = MockCompiledContractCache::new(0);
+    let cache: Option<&dyn CompiledContractCache> = Some(&mock_cache);
+    let vm_config = VMConfig::default();
+    let code1 = ContractCode::new(near_test_contracts::rs_contract().to_vec(), None);
+    let code2 = ContractCode::new(near_test_contracts::ts_contract().to_vec(), None);
+
+    let result = precompile_contract_impl(vm_kind, &code1, &vm_config, cache);
+    assert_eq!(result, Result::Ok(true));
```

Comment:
nit: here and elsewhere

```suggestion
    assert_eq!(result, Ok(true));
```